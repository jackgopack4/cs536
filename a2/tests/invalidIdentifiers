123_hahaha
$hello
~whatsthat
\hello
whatsup%

