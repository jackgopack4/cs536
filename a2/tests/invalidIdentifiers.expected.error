2:1 ***ERROR*** illegal character ignored: $
3:1 ***ERROR*** illegal character ignored: ~
4:1 ***ERROR*** illegal character ignored: \
5:8 ***ERROR*** illegal character ignored: %
