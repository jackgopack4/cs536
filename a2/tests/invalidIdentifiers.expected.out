Token: INTLITERAL
LineNum: 1
CharNum: 1
Value: 123

Token: ID
LineNum: 1
CharNum: 4
Value: _hahaha

Token: ID
LineNum: 2
CharNum: 2
Value: hello

Token: ID
LineNum: 3
CharNum: 2
Value: whatsthat

Token: ID
LineNum: 4
CharNum: 2
Value: hello

Token: ID
LineNum: 5
CharNum: 1
Value: whatsup

